canonical/canonical.com
# WD-13863 - add form template to c.com

## Done

Created a form template and necessary functions for its execution

REMOVE FORM FROM DATA OPENSEARCH AND POSTGRESQL AFTER TESTING

## QA

- Check the form on the demo: [static](https://canonical-com-1379.demos.haus/data/opensearch) and [dynamic](https://canonical-com-1379.demos.haus/data/postgresql#get-in-touch)
- Make sure that the form opens correctly and has all needed fields (you can check expected field sets in form-data.json)
- Check that the form submits necessary data
- Check that the form has correct required fields and does not submit if a required field is empty

## Issue / Card
[WD-13863](https://warthogs.atlassian.net/browse/WD-13863)

[WD-13863]: https://warthogs.atlassian.net/browse/WD-13863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
